ci-operator/step-registry/gather/must-gather/gather-must-gather-commands.sh

```diff
@@ -134,6 +136,71 @@ EOF
 EOF
     fi
   fi
+
+  # generate junit file for pre configuration steps failed
+  if test -f "${SHARED_DIR}/install-pre-config-status.txt" && [ "$(<"${SHARED_DIR}/install-pre-config-status.txt")" == "${EXIT_CODE_PRECONFIG}" ]
+  then
+    cat >"${ARTIFACT_DIR}/junit_install.xml" <<EOF
+      <testsuite name="cluster install" tests="2" failures="$INSTALL_POSTCHECK_FAILURE">
```

Comment:
`INSTALL_POSTCHECK_FAILURE` is not defined yet when this condition happened, right?

Reply:
right, should be 2 here, updated.